src/index.ts

```diff
@@ -223,27 +223,23 @@ export enum WebhookEventType {
   TransferOutStatusEvent = 'TransferOutStatusEvent',
 }
 
-export type WebhookRequestBody = {
-  eventType: WebhookEventType
-  provider: string
-  eventId: string
-  accountAddress: string
-}
-
-export type WebhookKycStatusRequestBody = WebhookRequestBody & {
```

Comment:
I removed these because I don't think these are used anywhere and also consistent with other similar types. But do we want to keep exporting these? E.g., 
```js
export type WebhookKycStatusRequestBody = WebhookRequestBody<WebhookEventType.KycStatusEvent]
```